src/test/java/uk/sky/cqlmigrate/CqlMigratorImplTest.java

```diff
@@ -258,14 +260,16 @@ public void schemaUpdatesTableShouldContainTheDateEachFileWasApplied() throws Ex
         //then
         ResultSet rs = session.execute("select * from schema_updates");
         for (Row row : rs) {
-            assertThat(row.getDate("applied_on")).as("applied_on").isNotNull().isAfter(now);
+            Date applied_on = row.getTimestamp("applied_on");
+            DateTime date = new DateTime(applied_on);
+            org.assertj.jodatime.api.Assertions.assertThat(date).as("applied_on").isNotNull().isAfter(now);
```

Comment:
Am I right in thinking this needed changing as the return type of `getDate` changed from `Date` to `LocalDate`?
This can be fixed by using the new method `getTimestamp` instead. E.g.:
```
    @Test
    public void schemaUpdatesTableShouldContainTheDateEachFileWasApplied() throws Exception {
        //given
        Date now = new Date();
        Collection<Path> cqlPaths = asList(getResourcePath("cql_valid_one"), getResourcePath("cql_valid_two"));

        //when
        MIGRATOR.migrate(CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);

        //then
        ResultSet rs = session.execute("select * from cqlmigrate_test.schema_updates");
        for (Row row : rs) {
            assertThat(row.getTimestamp("applied_on")).as("applied_on").isNotNull().isAfter(now);
        }
    }
```
Should be able to remove `joda-time` one that change is made.